ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -598,7 +604,16 @@ open class DrawerController: UIViewController {
     private func updatePreferredContentSize(isExpanded: Bool) {
         isPreferredContentSizeBeingChangedInternally = true
         if isExpanded {
-            preferredContentSize.height = UIScreen.main.bounds.height
+            let screenBound: CGFloat = UIScreen.main.bounds.height
+            if preferredMaximumExpansionHeight != -1 {
+                // Prferred max expansion height is not set.
+                preferredContentSize.height = screenBound
```

Comment:
**preferredContentSize.height = screenBound** is exactly the same in the first if block and in the else block.

Reply:
yeah , the aim is to reduce the number of checks  performed each time. Any suggestion on reorganising it better?

Reply:
My suggestion is to remove the first if block and have the condition of it covered so that the else is executed in that case.

You don't want to have the same instruction repeated on both blocks.

Reply:
made the change.